src/main/java/client/creator/MakeCharInfoHandler.java

```diff
@@ -0,0 +1,41 @@
+package client.creator;
+
+import client.Character;
+import provider.Data;
+import provider.DataProviderFactory;
+import provider.wz.WZFiles;
+
+public class MakeCharInfoHandler {
```

Comment:
Name it something other than "handler" to avoid confusion with packet handlers. Perhaps "validator"?